scripts/format_translations.py

```diff
@@ -136,6 +126,9 @@
 # Path to a user-generated file with a reference translation. User must generate and make available.
 REFERENCE_LOCALE_PATH = './en.json'
 
+# Attributes to be excluded from translation validation if needed.
+EXCLUDED_ATTRIBUTES = set(['consent.secondSection', 'consent.checkboxLabel', 'consent.title', 'consent.button.labelUnaccepted', 'consent.thirdSection', 'consent.firstSection', 'consent.versionHistory', 'flag.chooseLanguage'])
```

Comment:
Incredibly tiny note: sets are a pretty cool of feature in python. Did you know that you can actually write set literals? `val = {'item1', 'item2'}`

(they look a bit like dictionaries but without the `:value` part)